PowerEditor/src/WinControls/AboutDlg/AboutDlg.cpp

```diff
@@ -453,3 +453,93 @@ INT_PTR CALLBACK DoSaveOrNotBox::run_dlgProc(UINT message, WPARAM wParam, LPARAM
 	}
 	return FALSE;
 }
+
+
+void DoSaveAllBox::doDialog(bool isRTL)
+{
+
+	if (isRTL)
+	{
+		DLGTEMPLATE* pMyDlgTemplate = NULL;
+		HGLOBAL hMyDlgTemplate = makeRTLResource(IDD_DOSAVEALLBOX, &pMyDlgTemplate);
+		::DialogBoxIndirectParam(_hInst, pMyDlgTemplate, _hParent, dlgProc, reinterpret_cast<LPARAM>(this));
+		::GlobalFree(hMyDlgTemplate);
+	}
+	else
+		::DialogBoxParam(_hInst, MAKEINTRESOURCE(IDD_DOSAVEALLBOX), _hParent, dlgProc, reinterpret_cast<LPARAM>(this));
+}
+
+void DoSaveAllBox::changeLang()
+{
+	generic_string msg;
+	generic_string defaultMessage = TEXT("Are you sure you want to save all documents?\r\rChoose \"Always Yes\" if your answer will always be \"Yes\" and you won't be asked this question again.\rYou can re-activate this dialog in Preferences dialog later.");

```

Comment:
> Choose \"Always Yes\" if your answer will always be \"Yes\"

Don't you think it's a bit of a tautology?🙂
I would remove this text at all and show it as a tooltip.

Reply:
It's OK if it's simple & clear to users. it doesn't matter tautology to me. And tooltip is not necessary finally.

Reply:
But if you have a better text, feel free to provide.

Reply:
I would change it to something like this:
```
Choose "Always Yes" if you don't want to see this dialog again.
You can re-activate this dialog in Preferences later.
```
It is pretty clear what "Always Yes" does, so don't need to repeat it.